src/js/components/Table/Paginated/PaginatedTable.tsx

```diff
@@ -68,14 +68,12 @@ const propTypes = {
   noResultsText: PropTypes.string,
 };
 
-const Filter = (
+const SelectFilterType = (
```

Comment:
Small nitpick. We probably dont want to call this a `Type` since its quite a bit more than a type. I think `SelectFilter` is probably fine. Thoughts?